test/files/async_stuff.py

```diff
@@ -15,13 +15,13 @@
 
 
 async def go() -> None:
-    r = await async_session.scalars(text("select 1"))
+    r = await async_session.scalars(text("select 1"), params=[])
```

Comment:
It's the smallest change I could make that demonstrates that `params` is being validated correctly. Let me know if there are good execution parameters that should be used here instead.

Reply:
looks fine to me!